frontend/components/Course/StudentQueuePage/QuestionCard.tsx

```diff
@@ -47,6 +50,13 @@ const QuestionCard = (props: QuestionCardProps) => {
         });
     };
 
+    useEffect(() => {
+        if (question.status == QuestionStatus.ACTIVE) {
```

Comment:
Use "===" for better style, in fact this should probably be a lint error instead of warning